GoogleForCreators/web-stories-wp
# Add back "Set as background" button

## Summary
Adds back "Set as background" button to the Size & Position panel for media.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add Media
2. Open Size & Position panel
3. Verify there's a button "Set as background"
4. Verify it works as expected.

### UAT







This PR can be tested by following these steps:

1.





---



Fixes #7340
